services/powerbi/powerbi.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/adx-automation-services/services/powerbi/kubeutils"
+	"github.com/Azure/adx-automation-services/services/powerbi/logging"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+)
+
+type PowerBIService interface {
+	Refresh(context.Context, string, int64) error
+}
+
+type powerBIService struct{}
```

Comment:
Won't this throw error as it is not exported but used in another go package?

Reply:
This is not used in other packages